pkg/sqlite/migrator.go

```diff
@@ -0,0 +1,122 @@
+package sqlite
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database/sqlite3"
+	_ "github.com/golang-migrate/migrate/v4/source/file" // indirect import required by golang-migrate package
+)
+
+const (
+	// Hardcoded path where the db_migrations live
+	defaultMigrationsPath = "./pkg/sqlite/db_migrations"
```

Comment:
doesn't this mean that migrations only work if you have the `operator-registry` repo check out locally?

Reply:
Yep, that is true. Regardless you at the very least need the migrations folder. For the initializer, this is currently only run today from inside a container image that has the relevant files copied into it. We could also add a flag to the initializer that takes the migration folder as input, but in either case those files will need to be pulled down.

We could also, in theory, save the files themselves as source code so they could be compiled as part of the binary and then written to a directory before being read. That would make this more portable if we found that significantly valuable.

Reply:
A quick google led me to this: https://github.com/gobuffalo/packr - if this looks reasonable to add to this PR quickly (say <1hr?) can you go ahead and do that? I think the UX is much better if we don't have to ship migration files around.

If it looks like it would take longer than that, we don't have to bother with it now, and we can revisit the idea in the future.

Reply:
@ecordell I don't really like this tool because it makes a bunch of assumptions about the underlying filesystem of your files and also requires that you wrap `go build` inside their `packr` command. I just messed around with https://github.com/go-bindata/go-bindata and it seems like it's a little bit better for our specific use case -- it just generates a `.go` file with compressed binary versions of the files. We still need to use a binary tool to generate them, but we should be able to just add something to the makefile to download and run it when we need to generate the files.

I'm going to update this PR in a little bit with a second commit with an implementation that includes that.

Reply:
Awesome, thanks for finding that!